src/components/ContainerHeader.vue

```diff
@@ -13,8 +13,8 @@
             </div>
             <div class="kiwi-header-name">{{buffer.name}}</div>
             <div v-if="isJoined" class="kiwi-header-topic"></div>
-            <div v-if="!isJoined && isConnected" class="kiwi-header-notjoined">
-                <a @click="joinCurrentBuffer" class="u-link">{{$t('container_join')}}</a>
+            <div v-if="isJoined && isConnected" class="kiwi-header-notjoined">
+                <a @click="joinCurrentBuffer" class="u-link join-channel-button">{{$t('container_join')}}</a>
```

Comment:
note: class names like this should be prefixed with what section it's in. So here it should be `.kiwi-header-join-channel`